backtester/engine/live_test.go

```diff
@@ -0,0 +1,70 @@
+package engine
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/thrasher-corp/gocryptotrader/backtester/common"
+	"github.com/thrasher-corp/gocryptotrader/backtester/config"
+	gctexchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	gctkline "github.com/thrasher-corp/gocryptotrader/exchanges/kline"
+)
+
+func TestLoadLiveData(t *testing.T) {
+	t.Parallel()
+	err := loadLiveData(nil, nil)
+	if !errors.Is(err, common.ErrNilArguments) {
+		t.Error(err)
+	}
+	cfg := &config.Config{}
+	err = loadLiveData(cfg, nil)
+	if !errors.Is(err, common.ErrNilArguments) {
+		t.Error(err)
+	}
+	b := &gctexchange.Base{
+		Name: testExchange,
+		API: gctexchange.API{
+			AuthenticatedSupport:          false,
+			AuthenticatedWebsocketSupport: false,
+			PEMKeySupport:                 false,
+			CredentialsValidator: struct {
```

Comment:
Suggestion: make a struct? (don't have to do this.)

Reply:
Its more a result of `exchange_types.go` `type API struct`, I can modify it there to allow that here if you want.